packages/image_picker/image_picker_platform_interface/lib/src/platform_interface/image_picker_platform.dart

```diff
@@ -132,7 +132,11 @@ abstract class ImagePickerPlatform extends PlatformInterface {
   ///
   /// Use `preferredCameraDevice` to specify the camera to use when the `source` is [ImageSource.camera].
   /// The `preferredCameraDevice` is ignored when `source` is [ImageSource.gallery]. It is also ignored if the chosen camera is not supported on the device.
-  /// Defaults to [CameraDevice.rear].
+  /// Defaults to [CameraDevice.rear]. 
+  /// 
```

Comment:
Actually, I think you were right. It was related to the comment above. Whatever you did was correct. My mistake :) Can we revert this change? Sorry for the confusion.
Same above.

Reply:
No problem, will revert back